tests/evm.rs

```diff
@@ -5641,3 +5642,77 @@ fn refund_limit_value() {
 
     run_program_assert_gas_and_refund(env, db, needed_gas as _, used_gas as _, refunded_gas as _);
 }
+
+#[test]
+fn recursive_create() {
+    let value: u64 = 100000;
+    let sender_balance = EU256::from(20000000);
+    let sender_addr = Address::from_low_u64_be(5000);
+    let to_addr = Address::from_low_u64_be(3000);
+
+    let operations = vec![
+        Operation::Push((1, BigUint::from(32_u8))),
+        Operation::Push((1, BigUint::ZERO)),
+        Operation::Push((1, BigUint::ZERO)),
+        Operation::Codecopy,
+        Operation::Push((1, BigUint::from(32_u8))),
+        Operation::Push((1, BigUint::ZERO)),
+        Operation::Push((1, BigUint::ZERO)),
```

Comment:
We have a specific opcode for pushing zeros:
```suggestion
        Operation::Push0,
        Operation::Push0,
        Operation::Codecopy,
        Operation::Push((1, BigUint::from(32_u8))),
        Operation::Push0,
        Operation::Push0,
```

Reply:
Ah you are right, changing them